Emby.Server.Implementations/HttpServer/Security/AuthorizationContext.cs

```diff
@@ -111,82 +111,84 @@ private AuthorizationInfo GetAuthorization(HttpContext httpReq)
                 Token = token
             };
 
-            AuthenticationInfo originalAuthenticationInfo = null;
-            if (!string.IsNullOrWhiteSpace(token))
+            if (string.IsNullOrWhiteSpace(token))
             {
-                var result = _authRepo.Get(new AuthenticationInfoQuery
-                {
-                    AccessToken = token
-                });
+                // Request doesn't contain a token.
+                throw new SecurityException("Unauthorized.");
```

Comment:
This breaks most of the public endpoints. This code is called in the authenticationhandler, which is _always_ called.

Reply:
I tested public endpoints, it looks like the authentication handler is only called when the endpoint challenges it.

Reply:
You must have something weird going on then because it is always called even with AllowAnonymous. You're not supposed to deny access from AuthenticationHandler. It only creates claims etc.

Reply:
After more testing you're correct, but the exception is swallowed in the `CustomAuthenticationHandler`. I've moved the exception throwing to be near the other `SecurityException` for easier clean up in the future